scripts/compile_bundle.sh

```diff
@@ -99,44 +107,39 @@ git checkout ${VER_IPEX}
 git submodule sync
 git submodule update --init --recursive
 
-# Install the basic dependency cmake
-python -m pip install cmake
-
 # Compile individual component
 #  LLVM
 cd ../llvm-project
-git config --global --add safe.directory `pwd`
 if [ -d build ]; then
     rm -rf build
 fi
 mkdir build
 cd build
 cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-D_GLIBCXX_USE_CXX11_ABI=1" -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_TERMINFO=OFF -DLLVM_INCLUDE_TESTS=OFF -DLLVM_INCLUDE_EXAMPLES=OFF ../llvm/
-cmake --build . -j $(nproc)
-LLVM_ROOT=`pwd`/../release
+cmake --build . -j $MAX_JOBS
+LLVM_ROOT="$(pwd)/../release"
 if [ -d ${LLVM_ROOT} ]; then
 	rm -rf ${LLVM_ROOT}
 fi
 cmake -DCMAKE_INSTALL_PREFIX=${LLVM_ROOT}/../release/ -P cmake_install.cmake
-#xargs rm -rf < install_manifest.txt
 ln -s ${LLVM_ROOT}/bin/llvm-config ${LLVM_ROOT}/bin/llvm-config-13
 export PATH=${LLVM_ROOT}/bin:$PATH
 export LD_LIBRARY_PATH=${LLVM_ROOT}/lib:$LD_LIBRARY_PATH
 cd ..
-git config --global --unset safe.directory
 #  PyTorch
 cd ../pytorch
-git config --global --add safe.directory `pwd`
 git stash
 git clean -f
 git apply ../intel-extension-for-pytorch/torch_patches/*.patch
-python -m pip install astunparse numpy ninja pyyaml mkl-static mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses
 export USE_LLVM=${LLVM_ROOT}
 export LLVM_DIR=${USE_LLVM}/lib/cmake/llvm
-if [[ ${CONDA} -eq 0 ]]; then
-    export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
+if [ -n "${CONDA_PREFIX-}" ]; then
+    export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(command -v conda))/../"}
+elif [ -n "${VIRTUAL_ENV-}" ]; then
+    export CMAKE_PREFIX_PATH=${VIRTUAL_ENV:-"$(dirname $(command -v python))/../"}
 else
-    export CMAKE_PREFIX_PATH=${VIRTUAL_ENV:-"$(dirname $(which python))/../"}
+    # TODO not building with conda or virtualenv, what should be set?
+    continue
```

Comment:
Did you try running this script in a non-conda python environment?

Reply:
Yes, my builds and tests were inside a python virtual env.

https://github.com/intel/intel-extension-for-pytorch/blob/master/docker/Dockerfile.compile

Could you clarify why conda is added to the build, when it is not actually used as environment? 
I don't see a creation of a Conda environment in the Dockerfile?

i assume, the script needs to support three types of builds?

1. within a conda env
2. within a python virtual env
3. within a docker container (this is basically the "default" case without any env, both as root and normal user)

What path needs to be added to `CMAKE_PREFIX_PATH` in the 3rd case?

Thanks for all the feedback and consideration of my PR.

Reply:
It's OK to cover support of virtenv and normal python env.
For case 3, dependency python packages should be installed into /usr/lib? If so, no need to set CMAKE_PREFIX_PATH. The purpose of CMAKE_PREFIX_PATH is to make cmake able to find installed python packages in non-system search paths.